### PR TITLE
Updated documentation & examples, including settings.

### PR DESCRIPTION
**Pull request by Marcus10110**
Note - Logic 2.2.9 has not been released yet! It will be soon though.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Logic 2 Examples
 
+
+## API changelog
+
+Logic 2.2.6
+
+- Very first release of extensions, starting with High Level Analyzers!
+
+Logic 2.2.9
```
Comment (ranihorev) [inline README.md]: newer versions usually appear first, I think

File: README.md [hunk 2]
```diff
@@ -10,7 +22,31 @@ Saleae High Level Analyzers (HLAs) allow you to write python code that processes
...
 
 
-*HLAs require the Saleae Logic software version 2.2.6 or newer.*
+*HLAs require the Saleae Logic software version 2.2.6 or newer. HLA settings were introduced in 2.2.9*
+
+## Development Process
+
+Starting a new HLA is easy! First, install and open the latest version of the Logic software. Then, open the new extensions sidebar panel.
+
+From there, click the "+" button in the upper right. This will open the "Create an Extension" dialog. Here you can either create a new extension from a template, or load an existing extension from disk.
```
Comment (ranihorev) [inline README.md]: Could be nice to add a gif

File: README.md [hunk 3]
```diff
@@ -10,7 +22,31 @@ Saleae High Level Analyzers (HLAs) allow you to write python code that processes
...
+
+This will also load your extension into the software. To see it in action, capture some data that contains protocol data. (Serial, I2C, or SPI).
+
+Then, add the correct analyzer for your data (Serial, I2C, and SPI are supported). Once you've configured the built-in analyzer to decode the raw bytes of your capture, you can add your new HLA. From the same add analyzer menu, now locate your new HLA. It will probably be 4th in the list, right under Serial, I2C, and SPI.
+
+This will open the settings for your HLA. Right now there is only one, the input analyzer. There, you need to select the low-level analyzer you added earlier, Serial, I2C or SPI. Then press finish.
+
+The default HLA template will just copy the frames from the input analyzer to the output. Because it doesn't come with a format string (described below), the bubble text you will see on screen will probably look like this: '{type:"data", data:{value:10,...}}'. This conveniently shows you the data format of the input analyzer that you selected, which you will need to access in your HLA. The data format for your input analyzer is also documented below.
```
Comment (ranihorev) [inline README.md]: most people don't know what bubbles are :)

File: README.md (hunk 1 above)
Comment (Marcus10110) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (Marcus10110) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Marcus10110) [reply README.md]: done

**Pull request merged**